api/adobe_vendor_id.py

```diff
@@ -739,8 +739,11 @@ def from_config(cls, library, _db=None):
         ):
             raise CannotLoadConfiguration(
                 "Short Client Token configuration is incomplete. "
-                "vendor_id, username, password and "
-                "Library website_url must all be defined.")
+                "vendor_id (%s), username (%s), password (%s) and "
+                "Library website_url (%s) must all be defined." % (
+                    vendor_id, library_uri, library_short_name, secret
+                )
+            )
```

Comment:
This is an unrelated change that helps us recover more quickly from SIMPLY-2983 incidents -- with the current error message it's not clear which piece of configuration is missing.